### PR TITLE
release: bump version to 11.0.0-rc.2 w/ changelog

### PR DESCRIPTION
**Pull request by annieyw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
+# 11.0.0-rc.2 "vicuna-valley" (2020-11-10)
```
Comment (mmalerba) [inline CHANGELOG.md:1]: nit: looked up what a vicuna is and it appears to not be a material

Comment (annieyw) [reply CHANGELOG.md:1]: 🦙

Comment (mmalerba) [reply CHANGELOG.md:1]: TIL: a vicuna is an animal and also the name of the material made of its fur 🙂

**Review by mmalerba — APPROVED** [review]
LGTM

**Pull request merged**